skyvern/forge/sdk/forge_log.py

```
@@ -121,6 +122,154 @@ def __call__(self, logger: logging.Logger, name: str, event_dict: EventDict) ->
             return f"{file_section_colored} {rendered}"
 
 
+def add_error_processor(logger: logging.Logger, method_name: str, event_dict: EventDict) -> EventDict:
```

Ruff (0.14.8)

Unused function argument: `logger`

(ARG001)